src/app/core/utils/api-token/api-token.service.ts

```diff
@@ -45,9 +46,9 @@ import {
   loadUserByAPIToken,
 } from 'ish-core/store/customer/user';
 import { CookiesService } from 'ish-core/utils/cookies/cookies.service';
-import { mapToProperty, whenTruthy } from 'ish-core/utils/operators';
+import { whenTruthy } from 'ish-core/utils/operators';
 
-type ApiTokenCookieType = 'user' | 'order' | 'anonymous';
+type ApiTokenCookieType = 'user' | 'order';
```

Comment:
Please, add a short description what does this service do?, and maybe a link to the verbose docu